taggit-tacc/taggit
# Task/DES-2389: CSV/JSON Export

## Overview: ##
Fix the CSV/JSON export to work with new structure.

## Related Jira tickets: ##

* [DES-2389](https://jira.tacc.utexas.edu/browse/DES-2389)

## Testing Steps: ##
1. Create a group with images
2. Add Tag Templates
3. Save tags with tag structures
4. Ensure that all the saved tags show up properly in CSV/JSON
5. Ensure that saved tags are correctly aligned with the corresponding headers in CSV
